EIPS/eip-4966.md

```diff
@@ -0,0 +1,135 @@
+---
+eip: 4966
+title: Non-Transferrable Non-Fungible Tokens (Soulbound Tokens)
+description: A standard interface for non-transferrable non-fungible tokens, also known as "soulbound tokens" (short "SBT").
+author: Nicola Greco, Tim Daubenschütz (TimDaub)
+discussions-to: https://ethereum-magicians.org/t/eip-4966-non-transferrable-non-fungible-tokens-soulbound-tokens/8825
+type: Standards Track
+category: ERC
+status: Draft
+created: 2022-04-01
+requires: 165, 721
+---
+
+## Abstract
+
+The following standard allows for the implementation of a standard API for SBTs
+within smart contracts. This standard provides basic functionality to gift,
+mint and track SBTs.
+
+## Motivation
+
+[`ERC4966` tokens inhert their metadata model from [`ERC721`](./eip-721.md)'s
+metadata. Wallets can differentiate between `ERC4966` and transferrable
+[`ERC721`](./eip-721.md) tokens through [`ERC165`](./eip-165.md)'s
+`supportsInterface(bytes4 interfaceID);` method.
```

Comment:
Some weirdness with the link syntax and spelling.

```suggestion
`ERC4966` tokens inherit their metadata model from [`ERC721`](./eip-721.md)'s metadata. Wallets can differentiate between `ERC4966` and transferrable [`ERC721`](./eip-721.md) tokens through [`ERC165`](./eip-165.md)'s `supportsInterface(bytes4 interfaceID);` method.
```

This is also not really motivation content; it probably fits more in the Specification section. The Motivation section should describe why the EIP as a whole needs to exist.